spec.md

```diff
@@ -671,8 +671,9 @@ $ curl 'http://username:password@broker-url/v2/service_instances/:instance_id/
 
 | Status Code | Description |
 | --- | --- |
-| 200 OK | Binding was deleted. The expected response body is `{}`. |
-| 410 Gone | MUST be returned if the binding does not exist. The expected response body is `{}`. |
+| 200 OK | MUST be returned if the binding was deleted as a result of this request. The expected response body is `{}`. |
+| 404 Not Found | MUST be returned if the service binding does not exist. |
+| 410 Gone | MAY be returned if the binding is known to the service broker but had been previously deleted. The expected response body is `{}`. While a `410 Gone` is allowed, the RECOMMENDED response is `404 Not Found`. |
```

Comment:
Since a 404 may be temporarily returned by an intermediating component, we wouldn't want the platform to execute a state change based on it. Only 200 and 410 should be supported responses.